core/controllers/email_manager.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
This should go in core/domain -- it's not a controller.

Reply:
File relocated.